library/src/webapp/editor/ckextraplugins/common-wl/js/folder-listing.js

```diff
@@ -153,8 +154,4 @@ $.fn.folderListing = function(options) {
   });
 };
 
-// automatically bind to data-folder-listing divs
```

Comment:
I think this will stop the widgets working, although the widget isn't used anywhere else other than this now.